doma-spring-boot-core/src/main/java/org/seasar/doma/boot/UnifiedQueryPageable.java

```diff
@@ -0,0 +1,229 @@
+package org.seasar.doma.boot;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.seasar.doma.jdbc.criteria.declaration.OrderByNameDeclaration;
+import org.seasar.doma.jdbc.criteria.metamodel.EntityMetamodel;
+import org.seasar.doma.jdbc.criteria.metamodel.PropertyMetamodel;
+import org.seasar.doma.jdbc.criteria.statement.EntityQueryable;
+import org.springframework.data.domain.Pageable;
+
+/**
+ * An adapter that integrates {@link Pageable} with Doma Criteria API.
+ * <p>
+ * This class allows converting {@link Pageable} pagination and sort information
+ * into Doma Criteria API's limit, offset, and order-by specifications.
+ * <p>
+ * Example usage:
+ * <pre>{@code
+ * public Page<Task> getPage(Pageable pageable) {
+ *     final var task_ = new Task_();
+ *     final var p = UnifiedQueryPageable.from(pageable, task_);
+ *     final var content = this.queryDsl
+ *         .from(task_)
+ *         .offset(p.offset())
+ *         .limit(p.limit())
+ *         .orderBy(p.orderBy())
+ *         .fetch();
+ *     final var total = this.queryDsl
+ *         .from(task_)
+ *         .select(Expressions.count())
+ *         .fetchOne();
+ *     return new PageImpl<>(content, pageable, total);
+ * }</pre>
```

Comment:
It seems that the body block of the `getPage` method is not closed.

```suggestion
 * }
 * }</pre>
```

Reply:
Thank you! I didn't notice.